text/survey_analysis_report2.md

```diff
@@ -0,0 +1,191 @@
+
+![Fig. 2: Wilingness to use a template, split by preference for one template.](../50_figures/pulication_ready_figures/fig2.png)
+
+We asked three questions about where they would save or search for specific documents in the different structures.
+Researchers were indicating different folders between them, and different than the ones we designed for it,
+showing that **a too detailed folder structure would not be helpful**.
+On the other hand, researchers would navigate the repository to find specific files using similar strategies (Fig. 3),
+suggesting that having a structure can be helpful, and may reduce the time to browse for specific information.
+
+![Fig.3: Folder researchers would search for information about "blindness" status of the experiment design. Rank1 line shows that most people would look first in the main readme file or the preregistration folder.](../50_figures/pulication_ready_figures/fig3.png)
+
+## Issues
+
+Most participants could see the advantage of such an homogenization of the project management for their own work, but were critical about the **cost/benefit ratio** of such a process, especially for ongoing projects.
+Many mentioned that it would only reach full impact if the whole lab would be using it, emphasizing the advantage of such a system for
+collaborative work.
+In addition, they mentioned the time saved by not having to create a template for themselves,
+but only use an existing standard.
+While they were quite unanimous about using a template for new projects, they were critical about making a transition for ongoing projects,
+as the cost of transition might be too high.
+More generally, they were questioning the cost over benefit ratio of changing their current organization system,
+especially as learning a new workflow takes time.
+
+
+In many cases, people mentioned that the **files are organized via experiments**, not at the project level.
+In particular, people tend to pool data and code in a single experiment folder.
+This is reminiscent of the two example structure given in the library carpentry course [@LibraryCarpentryLcfairresearch2019].
+
+
+On the other hand, some **files are organised outside of project folders**, i.e.  in particular places irrespective of the project they belong to. 
+Many researchers reported having a folder for all conferences reports or all manuscritps, for instance.
+
+
+# The new template
+
+Data management principles recommend to keep all files related to a project in a single folder (this facilitates sharing of these files with the whole team working on the project),
+and to manage data and code differently (this allows different version control system, as well as sharing and reusing data and code independently).
+We designed a template that follows these principles, 
+but added some recommendations and technical solutions 
+in order to permit users to have cross-project and experiment-level organisations of their files.
```

Comment:
The definitions of "projects" and "experiments" sound unclear to me.
Weren't there some images, or a list of definitions, about what is a project, and what is an experiment?
I think it would be better to make it clear about the terminology here.

Reply:
very good feedback, I will look at the whole text to indeed get a definition of these terms (or an attempt at a definition).